ipn/ipnlocal/local.go

```diff
@@ -4685,14 +4713,7 @@ func (b *LocalBackend) applyPrefsToHostinfoLocked(hi *tailcfg.Hostinfo, prefs ip
 	hi.ShieldsUp = prefs.ShieldsUp()
 	hi.AllowsUpdate = envknob.AllowsRemoteUpdate() || prefs.AutoUpdate().Apply.EqualBool(true)
 
-	// count routes without exit node routes
-	var routes int64
-	for _, route := range hi.RoutableIPs {
-		if route.Bits() != 0 {
-			routes++
-		}
-	}
-	metricAdvertisedRoutes.Set(float64(routes))
+	b.metrics.advertisedRoutes.Set(float64(len(hi.RoutableIPs)))
```

Comment:
Hm, did we not need to exclude /0 routes here after all?

Reply:
So for both this and the previous ones, I ended up removing it. `/0` would be exit nodes, and they are a part of the advertised routes, so it would make sense to count them? Same for the other comment. Not sure what is the most correct.